client/me/connected-application-item/index.jsx

```diff
@@ -120,116 +118,100 @@ module.exports = React.createClass( {
-						<strong className="connected-application-item__connection-detail-title">
-							{ this.translate( 'Application Website' ) }
-						</strong>
-
-						<span className="connected-application-item__connection-detail-description">
-							<a
-								href={ safeProtocolUrl( connection.URL ) }
-								onClick={ this.recordClickEvent( 'Connected Application Website Link' ) }
-								target="_blank"
-							>
-								{ safeProtocolUrl( connection.URL ) }
-							</a>
-						</span>
-					</li>
-
-					<li className="connected-application-item__connection-detail">
-						{
-							this.translate( '{{detailTitle}}Authorized On{{/detailTitle}}{{detailDescription}}%(date)s{{/detailDescription}}', {
-								components: {
-									detailTitle: <strong className="connected-application-item__connection-detail-title" />,
-									detailDescription: <span className="connected-application-item__connection-detail-description" />
-								},
-								args: {
-									date: this.moment( connection.authorized ).format( 'MMM D, YYYY @ h:mm a' )
-								}
-							} )
-						}
-					</li>
 
+				{ this.translate( '{{detailTitle}}Authorized On{{/detailTitle}}{{detailDescription}}%(date)s{{/detailDescription}}', {
+					components: {
+						detailTitle: <h2 />,
+						detailDescription: <p className="connected-application-item__connection-detail-description" />
+					},
+					args: {
+						date: this.moment( this.props.connection.authorized ).format( 'MMM D, YYYY @ h:mm a' )
+					}
+				} ) }
+				<p>
 					{ this.renderScopeMessage() }
+				</p>
 
-					<li className="connected-application-item__connection-detail">
-						<strong className="connected-application-item__connection-detail-title">
-							{ this.translate( 'Access Permissions' ) }
-						</strong>
-
-						{ connection.permissions.map( function( permission ) {
-							return (
-								<span
-									className="connected-application-item__connection-detail-description"
-									key={ 'permission-' + permission.name } >
-									{ permission.description }
-								</span>
-							);
-						}, this ) }
-					</li>
+				<h2>
+					{ this.translate( 'Access Permissions' ) }
+				</h2>
+				<ul className="connected-application-item__connection-detail-descriptions">
+					{ this.props.connection.permissions.map( function( permission ) {
+						return (
+							<li key={ 'permission-' + permission.name } >
+								{ permission.description }
+							</li>
+						);
+					}, this ) }
 				</ul>
 			</div>
 		);
 	},
 
+	header: function() {
+		return (
+			<div className="connected-application-item__header">
+				<ConnectedApplicationIcon image={ this.props.connection.icon } />
+				<h3>{ this.props.connection.title }</h3>
+			</div>
+		);
+	},
+
+	summary: function() {
+		return( <div>{ this.props.isPlaceholder
+			? ( <Button compact disabled>{ this.translate( 'Loading…' ) }</Button> )
+			: ( <Button compact onClick={ this.disconnect }>{ this.translate( 'Disconnect' ) }</Button> ) }</div> );
```

Comment:
We should split this into multiple lines to match our style elsewhere and to make it explicitly clear:

```js
return (
	<div>
		{ isPlaceholder
			? <Button compact disabled>…</button>
			: <Button compact onClick=…>…</button> }
	</div>
);
```